### ISSUE TITLE
Add container letsencrypt to docker-compose.yml

### ISSUE THREAD

**Issue by 4n70w4** [docker-taiga/taiga#6]

**Comment by w1ck3dg0ph3r** [comment]
Good idea, but I'm planning on merging back, front and events images to reduce the number of hops to reach each component (with single nginx instance running inside of it). When it is done, I will add an option to use external SSL termination, user provided certificates or automatically issuing letsencrypt's ones.

**Comment by 4n70w4** [comment]
May be a better solution would be to use traefik as reverse proxy which out of the box creates and automatically prolong lets encript certificates. But I could not connect to the traefik with this docker composer stack. Maybe something with network configuration.

**Comment by ArvsIndrarys** [comment]
Another option would be [caddy](https://github.com/abiosoft/caddy-docker). I use it on some of my projects and it is for me the easiest to install + the image is lightweight.

**Comment by 4n70w4** [comment]
May be it is not necessary. It is better to use the https://github.com/containous/traefik as a balancer, which itself automatically generates and renews letsencrypt certificates.

**Comment by marshalleq** [comment]
I've been successfully using this container for letsencrypt proxying (based on nginx) for Taiga (and many other apps) for some time now.  https://docs.linuxserver.io/general/swag

It worked by generating the required certificates and forwarding 443 or your chosen port to other ports within a docker net or host net.  It's quite simple, just gotta get your head around it.  Definitely the best all in one docker container for this I've found.
